### PR TITLE
Fix invalid string literal

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Run.hs [hunk 1]
```diff
@@ -182,7 +185,7 @@ checkLiveViewrequiredTracers traceConfig = do
   if all (== True) reqTracers
   then pure ()
   else do putTextLn "for full functional 'LiveView', please turn on the following \
-                    \tracers in the configuration file: TraceBlockFetchDecisions, \\
+                    \tracers in the configuration file: TraceBlockFetchDecisions, \
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Node/Run.hs:188]: This is the substantive fix.

File: cardano-node/src/Cardano/Node/Run.hs [hunk 2]
```diff
@@ -73,18 +74,20 @@ import qualified Ouroboros.Consensus.Storage.ChainDB as ChainDB
...
 
+import           Cardano.Node.Configuration.Socket (SocketOrSocketInfo (..),
+                     gatherConfiguredSockets)
+import           Cardano.Node.Configuration.Topology
+import           Cardano.Node.Handlers.Shutdown
 import           Cardano.Node.Protocol (SomeConsensusProtocol (..), mkConsensusProtocol,
                      renderProtocolInstantiationError)
-import           Cardano.Node.Handlers.Shutdown
-import           Cardano.Node.Configuration.Socket (SocketOrSocketInfo (..), gatherConfiguredSockets)
-import           Cardano.Node.Configuration.Topology
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Node/Run.hs:80]: Our coding style currently allows for `100` characters per line, but we're committing code past this.  Should we extend the line limit to `120`

Comment (Jimbo4350) [reply cardano-node/src/Cardano/Node/Run.hs:80]: I vote to leave the width at 100 chars

Comment (mrBliss) [reply cardano-node/src/Cardano/Node/Run.hs:80]: As a sucker for code style/formatting, I suggest letting CI enforce the use of `stylish-haskell`, like we do in consensus (https://github.com/input-output-hk/ouroboros-network/blob/7454eaca85e3ffc298e973767316ab2eac2bbe7b/.buildkite/pipeline.yml#L12). This repository is touched by too many different people who are not aware of the style conventions (I'm guilty myself!). Although I think not everybody will agree with making `stylish-haskell` mandatory :grin:.

Comment (intricate) [reply cardano-node/src/Cardano/Node/Run.hs:80]: I agree with @mrBliss

File: cardano-node/src/Cardano/Node/Run.hs (hunk 1 above)
Comment (newhoggy) [reply cardano-node/src/Cardano/Node/Run.hs]: I moved the above function to its own module to avoid `CPP` language extension interference.

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**